notebooks/02-Variables-Operators-Types.ipynb

```diff
@@ -0,0 +1,1741 @@
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "It's a good idea to take advantage of this flexibility and name your variables with descriptions that help you remember what they contain."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "For example, calling your variables `x`, `y`, and `z` is likely to lead to forgetting what you've stored where (unless you're working with coordinates, a domain where those names have meanings)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "More descriptive names, like `number_of_items` or `size_of_container`, are better."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Perfectly good variable name\n",
+    "my_3rd_favorite_number = 18"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Legal, but undescriptive, variable name\n",
+    "a = 7"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": [
+     "ci-skip"
+    ]
+   },
+   "outputs": [
+    {
+     "ename": "SyntaxError",
+     "evalue": "invalid decimal literal (1815327299.py, line 2)",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;36m  Cell \u001b[0;32mIn[10], line 2\u001b[0;36m\u001b[0m\n\u001b[0;31m    4_plus_1 = 4 + 1\u001b[0m\n\u001b[0m     ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid decimal literal\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Illegal variable name -- it starts with a number\n",
+    "4_plus_1 = 4 + 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "If you try to name a variable something illegal, Python will gently remind you to follow the rules with a `SyntaxError` and an arrow indicating the location of the error."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "<div class=\"admonition warning alert alert-danger\">\n",
+    " <b><p class=\"first admonition-title\" style=\"font-weight: bold\">Caution!</p></b>\n",
+    " <p>Sometimes Python doesn't pinpoint the error very well, and the error will not be in the same place as the arrow.</p>\n",
+    "</div>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Your Turn\n",
```

Comment:
Up to you: most of these "Your Turn" headers in other notebooks have been `h3`, with `h2` as a section divider. Not a huge deal though.

Reply:
They do say that inconsistency is the hobgoblin of sloppy teachers. I'll make a quick fix.

Reply:
> They do say that inconsistency is the hobgoblin of sloppy teachers.

Can this be added to a style guide somewhere?

Reply:
I am about 95% sure that it's actually in the original Code of Hammurabi, I should have consulted that before I dove into making these slides.

Reply:
Fixed in [a706749](https://github.com/uc-python/python-training-cincinnati-insurance/pull/33/commits/a7067491459c1ba9392970d8ebe24c1a35669a1b)